velox/connectors/hive/HiveDataSink.cpp

```diff
@@ -971,8 +974,14 @@ uint64_t HiveDataSink::WriterReclaimer::reclaim(
 
   const uint64_t memoryUsageBeforeReclaim = pool->currentBytes();
   const std::string memoryUsageTreeBeforeReclaim = pool->treeMemoryUsage();
+  const auto writtenBytesBeforeReclaim = ioStats_->rawBytesWritten();
   const auto reclaimedBytes =
       exec::MemoryReclaimer::reclaim(pool, targetBytes, maxWaitMs, stats);
+  addThreadLocalRuntimeStat(
+      "earlyFlushedRawBytes",
+      RuntimeCounter(
+          ioStats_->rawBytesWritten() - writtenBytesBeforeReclaim,
+          RuntimeCounter::Unit::kBytes));
```

Comment:
Can we add two metrics here? one is early flushed bytes as SUM and the other early flushed count as SUM as well? Thanks!